ui/src/store/serializers.ts

```diff
@@ -83,8 +83,13 @@ export function serializeDimensionMetadata (dimension: DimensionMetadata): Dimen
 export function serializeJobCollection (collection: JobCollection): JobCollection {
   return Object.freeze({
     ...serializeResource(collection),
+    actions: {
+      ...serializeActions(collection.actions),
+      createTransform: collection.actions.createTransform,
+      createPublish: collection.actions.createPublish,
+    },
     member: collection.member.map(Object.freeze),
-  }) as JobCollection
+  }) as unknown as JobCollection
```

Comment:
Why was `unknown` needed here now?

Reply:
@martinmaillard I let you answer this :wink:

Reply:
It was telling me that the intersection between both types was not sufficient to just use `as ...` (because of all the missing properties of Collection).